### DISCUSSION TITLE
Non-admin usage.

### DISCUSSION THREAD

**Discussion by AlanEdd** [Xanashi/Icaros#91] [Q&A] [answered]
Hey there, 
I am aware that this needs admin access as I have read all the documentation. I am not quite sure if this process was working on my machine at work (no admin). I got an Admin to install in my user account and he left open the settings so I set it up as needed. Perhaps it was working or maybe it wasn't and other apps were generating thumbnails. The admin would input his password so that I would use the config occassionally.

However I am not sure if Icaros uses autorun, or how that working in the background process works. When I restart my machine, I am not sure if it's actually running successfully.  I suppose I am asking about my logic above and is there a way to test if it's actually running in the background as in my case above... 
Thank you.
Alan.

**Comment by Xanashi** [comment, marked as answer]
Shell Extensions (like Icaros) needs administrative rights to be registered/installed, but as soon as they **are** registered, Windows Explorer will load them automatically whenever it needs them, and then unload them afterwards. 
Windows Explorer does not need administrative rights to load shell extensions, so once you've activated Icaros, you're good to go without any further admin involvement. At least not until you need to make changes in IcarosConfig again.

Best way to test if it's working, is just to download or copy some media files that you know Icaros should be able to thumbnail (MKV, OGM, etc.) and see if they actually thumbnail. In regards to Properties, just click any of your files that have properties enabled in Icaros,
and check in the File Explorer Details Pane, if the properties are loaded correctly.

**Reply by AlanEdd** [reply]
Great,
Thanks for the reply. I now understand. Should be usable, I think when the admin set the Icaros Config...we must have deactivated. I'll check next week as I have to wait for some time for each Admin issue to be tackled!

**Reply by Xanashi** [reply]
Great! If you have any more questions, don't hesitate to ask.
